hydrostats/ens_metrics.py

```diff
@@ -409,6 +409,17 @@ def ens_crps(obs, fcst_ens, adj=np.nan, remove_neg=False, remove_zero=False, llv
     return output
 
 
+def ens_crps_decomposition(obs, fcst_ens, adj=np.nan, remove_neg=False, remove_zero=False):
+
+    # Calculate the Psam quantity
+    Psam = fcst_ens - obs               # Difference the bins
+    Psam[Psam < 0] = 0                  # Apply the heavyside function
+    Psam = np.sum(Psam, axis=0)         # Aggregate across the ensembles
+
+    # Calculate the Pcli quantity
+
```

Comment:
This function looks like it was started and never finished

Reply:
Sorry, I intended to remove that. It's not needed since the existing Hersback function can be split to return the reliability term. I did another commit to the branch that should have removed it.